packages/components/src/components/sidebar-nav/sidebar-nav-collapsible.styles.ts

```diff
@@ -0,0 +1,33 @@
+import { JssStyle } from 'jss';
+
+export const styles: JssStyle = {
+  collapsible: {
+    margin: 0,
+  },
+  heading: {
```

Comment:
So far we have used `BEM` style for classes - we need to discuss if we want to continue using this approach. One benefit: With `BEM` we can export static `CSS` stylesheets which could be used without `web-components`